account_voucher_tax/account_voucher.py

```diff
@@ -401,8 +385,14 @@ def _preparate_move_line_tax(self, cr, uid, src_account_id,
         else:
             if reference_amount < 0:
                 debit_line_vals.pop('analytic_account_id')
+                debit_line_vals.update({
```

Comment:
It is not easy to see different between change by pylint and change to resolve the issue, I guess that this is the main change to resolver the issue

LGTM